code/game/objects/buckling.dm

```diff
@@ -63,6 +64,9 @@
 	if(MA && MA.buckled_to == src)
 		. = MA
 		MA.buckled_to = null
+		if(buckled_original_layer)
+			MA.layer = buckled_original_layer
```

Comment:
Should I change this to `MA.layer = initial(MA.layer)` and get rid of the variable and check? Mostly unsure of other things that may change the buckled atom's layer that we may want to preserve.

Reply:
leave it